Tiered maternal anemia intervention (#612)
ihmeuw/vivarium_research: docs/source/intervention_models/maternal_anemia_intervention/index.rst

 - the burden of the maternal anemia in terms of deaths and DALYs occurring in 
   the population at risk within each tier of the pyramid if we assume 
   intervention is not applicable; 
 - the impact at each tier on maternal anemia in terms of averted deaths and DALYs 
   if we assume complete pyramid of interventions are applied.

.. list-table:: Tiered Intervention
  :header-rows: 1

  * - Tier
    - Intervention
    - Product name
    - Product definition
    - Target population
    - Note
  * - 1
    - Large-scale food fortification
    - DFS
    - 
    - Women of reproductive age (15 to 49 years)
    - 
  * - 2
    - Supplementation
    - BEP
    - 
    - PLW at ANC service with BMI less than 18.5
    - 
  * - 2
    - Supplementation
    - MMS
    - 
    - PLW at ANC service with BMI greater than or equal to 18.5
    - 
  * - 3
    - Treatment
    - IV iron
    - 
    - Pregnant women in 2nd/3rd trimester with moderate/severe anemia referred 
      from ANC to higher level facility
    - 

.. todo::

  Fill out the table above with intervention product definition.

.. list-table:: Affected Outcomes
  :header-rows: 1

>   * - Outcome
>     - Effect
>     - Population
>     - Note
>   * - Population mean hemoglobin
>     - Additive
>     - non-PLW
>     - Effects are not stratified by intervention tiers
>   * - Population mean hemoglobin
>     - Additive and multiplicative combined
>     - PLW
>     - Effects are stratified by intervention tiers 

Reviewer: This table is a bit unclear to me. I made this suggestion because the interventions apply directly to hemoglobin shift (which is then associated with a multiplicative change in maternal mortality as a risk effect associated with the hemoglobin shift). Is that what you meant by "additive and multiplicative combined"?

```suggestion
  * - Outcome
    - Effect
    - Population
    - Note
  * - Population mean hemoglobin
    - Additive shift
    - Total population/PLW/anemic PLW
    - LSFF/supplementation/treatment effect sizes and target populations vary. Hemoglobin shift affects both anemia and maternal mortality 
```

Baseline Coverage Data
++++++++++++++++++++++

.. note::

>   No existing coverage of each intervention tier

Reviewer: I think there is existing coverage of iron supplementation among PLW. Perhaps link to the maternal supplementation page that has information on this.

Author: I don't have materials about the treatment coverage among target populations, can i find it out in J drive?

Reviewer: It's listed in the baseline coverage section here: https://vivarium-research.readthedocs.io/en/latest/intervention_models/targeted_bep_and_mms/index.html

I believe we may have ignored baseline coverage for this project at the time due to the rush nature, but we should consider it going forward!

Author: I see, thanks!

.. list-table:: Baseline coverage data
  :widths: 10 10 10 10 10
  :header-rows: 1

  * - Location
    - Subpopulation
    - Coverage parameter
    - Value
    - Note
  * - Bangladesh
    - 
    - 
    - 
    - 
  * - Malawi
    - 
    - 
    - 
    - 
  * - Nigeria
    - 
    - 
    - 
    - 
  * - Pakistan
    - 
    - 
    - 
    - 

Vivarium Modeling Strategy
--------------------------

> How to estimate maternal anemia burden in Vivarium:

Reviewer: I think we should be specific about which populations we're referring to here. The first tier of the intervention affects all women of reproductive age, not just PLW ("mothers").

 1. Construct population hemoglobin distributions for each location-/age- group;
 2. Apply pregnancy adjustment factor to hemoglobin distributions for pregnant 
    and general populations;

>  3. Modify hemoglobin distributions based on intervention risk effects;

Reviewer: ```suggestion
 3. Modify hemoglobin distributions based on intervention effects;
```

>  4. Calculate maternal anemia burden, measured by deaths, YLDS, and YLLs.

Reviewer: ```suggestion
 4. Calculate maternal anemia burden, measured by prevalence and YLDS.
 5. Calculate maternal disorders burden corresponding to the change in population mean hemoglobin levels among PLW
```

Reviewer: Actually, on second thought, we might not want to get into the effects of hemoglobin on anemia and maternal mortality on this page. I think that this page should focus just on the effect of the interventions on hemoglobin of the target populations (and how to identify the target populations). We already have documentation on the effects of hemoglobin here https://vivarium-research.readthedocs.io/en/latest/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/index.html

Hemoglobin

> ++++++++++

Reviewer: It will be helpful to link to the GBD 2019 hemoglobin exposure and effects pages that already exist!

Author: Ok, i can do that!

.. list-table:: Modeled Outcomes
  :widths: 10 10 10 10 10 10 10
  :header-rows: 1

  * - Outcome
    - Outcome type
    - Outcome ID
    - Affected measure
    - Effect size measure
    - Effect size
    - Note
  * - Hemoglobin
    - GBD risk
    - rei_92
    - Exposure prevalence

>     - Relative risk

Reviewer: Isn't this a shift?

Author: So for a risk effect in terms of shift, should we call it risk difference?

Reviewer: Yeah that sounds good

    - Detailed in table `Effect Size`
    - 

.. list-table:: Restrictions
  :widths: 10 10 10
  :header-rows: 1

  * - Restriction
    - Value
    - Note
  * - Male only
    - False
    - 
  * - Female only
    - True
    - 
  * - Age group start
    - 15 years
    - 
  * - Age group end
    - 49 years

>     - 

Reviewer: I think GBD definition of reproductive age is actually 10 to 54. We trimmed it for this project by request of BMGF, but we probably want to keep it consistent with GBD otherwise.
